__tests__/ApplicationTest.js

```diff
@@ -32,12 +32,102 @@ describe("문자열 계산기", () => {
     });
   });
 
-  test("예외 테스트", async () => {
-    const inputs = ["-1,2,3"];
+  test('커스텀 구분자 사용', async () => {
+    const inputs = ['//;\\n1'];
+    mockQuestions(inputs);
+
+    const logSpy = getLogSpy();
+    const outputs = ['결과 : 1'];
+
+    const app = new App();
+    await app.run();
+
+    outputs.forEach((output) => {
+      expect(logSpy).toHaveBeenCalledWith(expect.stringContaining(output));
+    });
+  });
+
+  test('커스텀 구분자로 분리', async () => {
+    const inputs = ['//;\\n1;2;3'];
+    mockQuestions(inputs);
+
+    const logSpy = getLogSpy();
+    const outputs = ['결과 : 6'];
+
+    const app = new App();
+    await app.run();
+
+    outputs.forEach((output) => {
+      expect(logSpy).toHaveBeenCalledWith(expect.stringContaining(output));
+    });
+  });
+
+  test('커스텀 구분자에 숫자가 들어갔을 때', async () => {
+    const inputs = ['//123\\n412351236'];
+    mockQuestions(inputs);
+
+    const logSpy = getLogSpy();
+    const outputs = ['결과 : 15'];
+
+    const app = new App();
+    await app.run();
+
+    outputs.forEach((output) => {
+      expect(logSpy).toHaveBeenCalledWith(expect.stringContaining(output));
+    });
+  });
+
+  test('예외 테스트', async () => {
+    const inputs = ['-1,2,3'];
+    mockQuestions(inputs);
+
+    const app = new App();
+
+    await expect(app.run()).rejects.toThrow('[ERROR]');
```

Comment:
`toThrow`에 문자열을 전달하면 해당 문자열이 에러 메시지에 "포함"되는지를 확인한다고 합니다.
아래처럼 정규식을 이용해서 에러 메시지가 [ERROR] 로 시작하는지를 좀 더 구체적으로 확인해 볼 수 있을 것 같아요. ㅎㅎ
```suggestion
    await expect(app.run()).rejects.toThrow(/^\[ERROR\]/);
```
https://jestjs.io/docs/expect#tothrowerror

Reply:
좋은 피드백 감사합니다! 사실 이 부분은 원래 과제 템플릿에 내재되어있던 부분이라 일부러 건들지 않은 부분도 있었습니당 ㅎㅎ 근데 저렇게 정규식으로 바꾸는게 훨씬 더 정확하겠네요 !